sklearn/metrics/scorer.py

```diff
@@ -564,6 +605,7 @@ def make_scorer(score_func, greater_is_better=True, needs_proba=False,
                average_precision=average_precision_scorer,
                neg_log_loss=neg_log_loss_scorer,
                brier_score_loss=brier_score_loss_scorer,
+               neg_brier_score_loss=neg_brier_score_scorer,
```

Comment:
this should be ``neg_brier_score_loss=neg_brier_score_loss_scorer``
Maybe my comment is not clear enough, sorry.